Is there anything I can do to fix the failing builds ? Can you retry them maybe ?

Or maybe this need to be fixed, but I'm not sure why it's happening only on the windows builds

```
    def test_rpc_trade_history(mocker, default_conf, markets, fee):
        mocker.patch('freqtrade.rpc.telegram.Telegram', MagicMock())
        mocker.patch.multiple(
            'freqtrade.exchange.Exchange',
            markets=PropertyMock(return_value=markets)
        )
    
        freqtradebot = get_patched_freqtradebot(mocker, default_conf)
        create_mock_trades(fee)
        rpc = RPC(freqtradebot)
        rpc._fiat_converter = CryptoToFiatConverter()
        trades = rpc._rpc_trade_history(2)
        assert len(trades['trades']) == 2
        assert trades['trades_count'] == 2
        assert isinstance(trades['trades'][0], dict)
        assert isinstance(trades['trades'][1], dict)
    
        trades = rpc._rpc_trade_history(0)
        assert len(trades['trades']) == 2
        assert trades['trades_count'] == 2
        # The first closed trade is for ETC ... sorting is descending
>       assert trades['trades'][-1]['pair'] == 'ETC/BTC'
E       AssertionError: assert 'XRP/BTC' == 'ETC/BTC'
E         - ETC/BTC
E         + XRP/BTC

tests\rpc\test_rpc.py:298: AssertionError
```

It looks like the problem might be coming from the sorting of the list on windows, as it looks like we don't have this sorting problem on Linux.

```
(.env) # python3.7 .env/bin/py.test tests/rpc/test_rpc.py 
========== test session starts ==========
platform linux -- Python 3.7.5, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
Test order randomisation NOT enabled. Enable with --random-order or --random-order-bucket=<bucket_type>
rootdir: /home/thomas/github/Th0masL/freqtrade
plugins: asyncio-0.14.0, random-order-1.0.4, cov-2.11.1, mock-3.5.1
collected 24 items

tests/rpc/test_rpc.py ........................                        [100%]

========== 24 passed, 2 warnings in 0.96s ==========
```

--- a/tests/conftest_trades.py
+++ b/tests/conftest_trades.py
@@ -88,7 +88,7 @@ def mock_trade_2(fee):
         timeframe=5,
         sell_reason='sell_signal',
         open_date=datetime.now(tz=timezone.utc) - timedelta(minutes=20),
-        close_date=datetime.now(tz=timezone.utc),
+        close_date=datetime.now(tz=timezone.utc) - timedelta(minutes=2),
     )
     o = Order.parse_from_ccxt_object(mock_order_2(), 'ETC/BTC', 'buy')
     trade.orders.append(o)

